bundle/manifests/node-observability-operator-controller-role_rbac.authorization.k8s.io_v1_clusterrole.yaml

```diff
@@ -0,0 +1,57 @@
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRole
+metadata:
+  creationTimestamp: null
+  labels:
+    scc: node-observability-scc
+  name: node-observability-operator-controller-role
```

Comment:
Thoughts on https://github.com/openshift/node-observability-operator/pull/90#discussion_r976435550? A few other things need to be renamed as well.
This does break backward compatibility in terms of clean-up.

Reply:
Indeed, this is not very clean as we leave some leftover for the installations which have already been made.
However keeping misleading or not optimal naming is not clean either.
Since we are in TechPreview and I doubt the big number of customers for this operator, I'd say let's make the clear naming. Also, if we will do so for all the 3 (service account, cluster role and cluster role binding), this will have the least possible impact as the cluster role binding will be pointing to a not existing service account.